actor/src/main/scala/org/apache/pekko/actor/Scheduler.scala

```diff
@@ -230,6 +230,29 @@ trait Scheduler {
       implicit executor: ExecutionContext): Cancellable =
     schedule(initialDelay, interval, runnable)(executor)
 
+  /**
+   * Deprecated API: See [[Scheduler#scheduleWithFixedDelay]] or [[Scheduler#scheduleAtFixedRate]].
+   */
+  @deprecated(
+    "Use scheduleWithFixedDelay or scheduleAtFixedRate instead. This has the same semantics as " +
+    "scheduleAtFixedRate, but scheduleWithFixedDelay is often preferred.",
+    since = "Akka 2.6.0")
+  @nowarn("msg=deprecated")
```

Comment:
this meathod should be removed too

Reply:
* this method is actually used so I plan to come back to it again in another PR
* this PR is big enough already